git-guides/git-overview.md

```diff
@@ -0,0 +1,107 @@
+
+If you're getting started with Git, a great place to start is the [Git Cheat sheet](https://github.github.io/training-kit/). It's translated into many languages, [open source as a part of the `github/training-kit` repository](https://github.com/github/training-kit), and a great starting place for the fundamentals on the command line.
+
+<!--- Very simplified breakdown of the most basic commands with clear navigation to learn more about each one (this might be a good example).-->
+Some of the most important and most used commands that you'll find there are:
+
+- `git clone [url]`: Clone (download) a repository that already exists on GitHub, including all of the files, branches, and commits.
+- `git status`: Always a good idea, this command shows you what branch you're on, what files are in the working or staging directory, and any other important information.
+- `git branch`: This shows the existing branches in your local repository. You can also use `git branch [banch-name]` to create a branch from your current location, or `git branch --all` to see all branches, both the local ones on your machine, and the remote tracking branches stored from the last `git pull` or `git fetch` from the remote.
+- `git checkout [branch-name]`: Switches to the specified branch and updates the working directory.
+- `git add [file]`: Snapshots the file in preparation for versioning, adding it to the staging area.
+- `git commit -m "descriptive message"`: Records file snapshots permanently in version history.
+- `git pull`: Updates your current local working branch with all new commits from the corresponding remote branch on GitHub. `git pull` is a combination of `git fetch` and `git merge`.
+- `git push`: Uploads all local branch commits to the remote.
+  
+<!-- Again, potentially include illustrations from individual command pages here or more general learning illustrations. -->
+
+If you're looking for more GitHub specific technical guidance, check out [GitHub's help documentation](https://help.github.com/).
+
+### Getting Started With GitHub
+
+If you're wondering where Git ends and GitHub begins, you're not alone. They are tied closely together to make working with them both a seamless experience. While Git takes care of the underlying version control, GitHub is the collaboration platform built on top of it. GitHub is the place for pull requests, comments, reviews, integrated tests, and so much more. Most developers work locally to develop, and use GitHub for collaboration. That ranges from using GitHub to host the shared remote repository, to working with colleagues and capitalizing on features like protected branches, code review, GitHub Actions, and more.
+
+The best place to practice using Git and GitHub is the [Introduction to GitHub Learning Lab course](https://lab.github.com/githubtraining/introduction-to-github).
+
+If you already know Git and need to sign up for a GitHub account, head over to [github.com](https://github.com/).
```

Comment:
@brianamarie one thing we could add to the page is more "getting started with Git" content that is agnostic of GitHub (how to set up a repo locally etc.) but it might make sense to come back to this overview page when we have some of the other pages ironed out.